examples/chip-tool/commands/clusters/ClusterCommand.h

```diff
@@ -109,6 +118,11 @@ class ClusterCommand : public InteractionModelCommands, public ModelCommand, pub
                 return;
             }
         }
+        if ((path.mEndpointId == chip::kRootEndpointId) && (path.mClusterId == chip::app::Clusters::IcdManagement::Id) &&
+            (path.mCommandId == chip::app::Clusters::IcdManagement::Commands::UnregisterClient::Id))
+        {
+            ModelCommand::ClearICDEntry(mScopedNodeId);
```

Comment:
Why does this need the `ModelCommand` prefix?  It shouldn't.

Reply:
https://github.com/project-chip/connectedhomeip/pull/33690
follow-up here